drivers/sensor/icm42688/Kconfig

```diff
@@ -4,6 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+DT_COMPAT_ICM42688 := invensense,icm42688
```

Comment:
Most (all?) of these have been removed. Is this a rebasing mistake?

Reply:
Almost certainly